build_image.py

```diff
@@ -194,6 +194,8 @@ def create_dockerfile_base(config: OfrakImageConfig) -> str:
 
     dockerfile_base_parts += [
         "FROM python:3.7-bullseye@sha256:338ead05c1a0aa8bd8fcba8e4dbbe2afd0283b4732fd30cf9b3bfcfcbc4affab",
+        "RUN apt-get -y update",
```

Comment:
You'll need to install `pre-commit` to run the pre-commit hooks: https://ofrak.com/docs/contributor-guide/getting-started.html#pre-commit.

Reply:
Yes, thanks, saw that afterwards.